packages/odyssey-storybook/src/components/odyssey-mui/Status/Status.mdx

```diff
@@ -7,47 +7,63 @@ import * as Stories from "./Status.stories";
 # Status
 
 <Canvas>
-  <Story of={Stories.Success} />
+  <Story of={Stories.SuccessLamp} />
 </Canvas>
 
 ## Behavior
 
 Status is not an interactive element. It is only intended to inform. If a Status indicates that a user should take action, be sure to provide the appropriate controls.
 
-## Variants
+## States
 
-There are four variants of Status available: Default, Success, Caution, and Danger.
+There are four states of Status available: Default, Success, Caution, and Danger. Each of these states is available in two indicator styles: **lamp** and **pill**.
```

Comment:
What's the difference between lamp and pill?

Reply:
<img width="203" alt="Screenshot 2023-06-05 at 1 33 10 PM" src="https://github.com/okta/odyssey/assets/36284167/bc86dc12-745c-4222-9145-deb23c8fd961">

It's a purely visual difference to give product designers more options. We want them to default to `lamp` though.